dotcom-rendering/src/components/ArticleContainer.tsx

```diff
@@ -162,7 +162,8 @@ const adStyles = css`
 		constants.AD_LABEL_HEIGHT}px;
 
 		${from.desktop} {
-			min-height: ${adSizes.mpu.height + constants.AD_LABEL_HEIGHT}px;
+			min-height: ${adSizes.outstreamDesktop.height +
```

Comment:
This change will affect all desktop inline slots, instead of just the slot where Teads can render. However I don't think this matters, since on desktop inline2+ slots render in the right-hand column. 

If you wanted to make it more specific and perhaps a bit more futureproof, you could target the inline1 slot specifically (where Teads can render) to have this larger minimum height.

Reply:
It is better to have it more specific and futureproof Dom. Is this better f33bfd5d6bfe7043352f38b2f42f1a3abf8993a7?

Reply:
Yeah I think this makes sense. It would also be good to specifically target only the `inline1` slots that have passed back, i.e. the slot is a child `div` with an id of `dfp-ad--inline1--passback`.

Reply:
As mentioned in our call we will just target `dfp-ad--inline1` with placeholder height that we want and add a comment to make sure this value will be revisited before deciding to serve OPT-OUT for inline1. @domlander